packages/host/app/components/operator-mode/code-submode.gts

```diff
@@ -97,21 +96,22 @@ type SelectedAccordionItem = 'schema-editor' | null;
 
 const CodeModePanelWidths = 'code-mode-panel-widths';
 const defaultLeftPanelWidth =
-  (14.0 * parseFloat(getComputedStyle(document.documentElement).fontSize)) /
-  (document.documentElement.clientWidth - 40 - 36);
+  ((14.0 * parseFloat(getComputedStyle(document.documentElement).fontSize)) /
+    (document.documentElement.clientWidth - 40 - 36)) *
+  100;
 const defaultPanelWidths: PanelWidths = {
   // 14rem as a fraction of the layout width
   leftPanel: defaultLeftPanelWidth,
-  codeEditorPanel: (1 - defaultLeftPanelWidth) / 2,
-  rightPanel: (1 - defaultLeftPanelWidth) / 2,
-  emptyCodeModePanel: 1 - defaultLeftPanelWidth,
+  codeEditorPanel: (100 - defaultLeftPanelWidth) / 2,
+  rightPanel: (100 - defaultLeftPanelWidth) / 2,
+  emptyCodeModePanel: 100 - defaultLeftPanelWidth,
 };
 
 const CodeModePanelHeights = 'code-mode-panel-heights';
 const ApproximateRecentPanelDefaultFraction =
```

Comment:
If we're going to make this out of 100, rename the constant to `ApproximateRecentPanelDefaultPercentage`